packages/test-runner-commands/browser/commands.mjs

```diff
@@ -110,9 +111,19 @@ export async function getSnapshotConfig() {
   return snapshotConfig;
 }
 
-export async function getSnapshots() {
-  if (snapshots) {
-    return snapshots;
+const ESCAPE_REGEX = /snapshots\[[^\]]+] = (\n)?(?<content>`[^`]+`)/gm;
```

Comment:
A little comment of what this regex achieves and I'm good here!

Reply:
done!